docs/tutorials/mongoid-configuration.txt

```diff
@@ -562,6 +562,42 @@ instance does have an associated time zone, and this time zone will be used
 even if it is different from ActiveSupport's configured time zone when
 ``use_activesupport_time_zone`` is true.
 
+Configuring ``SSLContext``
+==========================
+It may be desirable to further configure TLS options in your application, for
+example by enabling or disabling certain ciphers.
+
+This can be done by setting TLS context hooks on the Ruby driver -- TLS context
+hooks are user-provided ``Proc``s that will be invoked before any TLS socket
+connection in the driver and can be used to modify the underlying
+``OpenSSL::SSL::SSLContext`` object used by the socket.
+
+To set TLS context hooks, add ``Proc``s to the ``Mongo.tls_context_hooks``
+array. This can be done in an initializer. The example below adds a hook
+that only enables the "AES256-SHA" cipher.
+
+.. code-block:: ruby
+
+  Mongo.tls_context_hooks.push(
+    Proc.new { |context|
+      context.ciphers = ["AES256-SHA"]
+    }
+  )
+
+  # Only the AES256-SHA cipher will be enabled from this point forward
+
+Every ``Proc`` in ``Mongo.tls_context_hooks`` will be passed an
+``OpenSSL::SSL::SSLContext`` object as its sole argument. These ``Proc``s will
+be executed sequentially during socket creation.
+
+..warning ::
+
+  TLS context hooks are global and will affect all ``Mongo::Client`` instances
+  in an application.
+
+For more information about TLS context hooks, including best practices for
+assigning and removing them, see `the Ruby driver documentation <https://docs.mongodb.com/ruby-driver/current/tutorials/ruby-driver-create-client/#modifying-sslcontext>`_.
```

Comment:
Let me know what you think of this level of specificity -- I wanted to give the user an overview while not copy and pasting everything from the driver documentation, since that seems harder to maintain. I'm happy to bring over all the information if you think that's more useful though.

Reply:
This is great, definitely a sufficient amount of info for Mongoid.